### DISCUSSION TITLE
Working with model 1.5

### DISCUSSION THREAD

**Discussion by mahbub2649** [basujindal/stable-diffusion#210] [Q&A]
How to make it work with 1.5?
It gets stuck in loading the model.

**Comment by cansakirt** [comment]
It's not easy to answer a question like yours. [This checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) should be helpful to you in future.

Have you tried using the --ckpt argument to point to the v1-5 model? ` --ckpt /path/to/your/model/v1-5-pruned-emaonly.ckpt`
